saleor/graphql/payment/tests/mutations/test_transaction_request_action.py

```diff
@@ -122,6 +122,13 @@ def test_transaction_request_charge_action_for_order(
     assert Decimal(event.parameters["amount"]) == expected_called_charge_amount
     assert event.parameters["reference"] == transaction.psp_reference
 
+    assert TransactionEvent.objects.get(
```

Comment:
IMO we should use `exists` instead of `get` in tests as if it the objects doesn't exist we will get the error from the `get` method instead of the assertion error.

Reply:
The difference is, that I'm checking if there is only one record with get()